ibis/pandas/core.py

```diff
@@ -142,7 +154,9 @@ def is_computable_input_arg(arg):
 )
 
 
-def execute_with_scope(expr, scope, aggcontext=None, clients=None, **kwargs):
+def execute_with_scope(
+    expr, scope, timecontext=None, aggcontext=None, clients=None, **kwargs
```

Comment:
can you type timecontext at least in all signatures

Reply:
Yes, typed them all.